packages/errors/package.json

```diff
@@ -43,6 +43,7 @@
     "@endo/ses-ava": "^1.1.1",
     "ava": "^5.3.0",
     "c8": "^7.14.0",
+    "ses0_18_3": "npm:ses@0.18.3",
```

Comment:
OMFG

Reply:
This is, incidentally, one of the tools I’ve intimated to @warner and @mhofman that we have available to us should it ever be necessary to depend on multiple versions of `xsnap`.

Reply:
Thx. Resolving.